modules/trino/src/test/java/org/testcontainers/containers/TrinoContainerTest.java

```diff
@@ -0,0 +1,122 @@
+package org.testcontainers.containers;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.testcontainers.TrinoTestImages;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.lang.Integer.parseInt;
+import static java.lang.String.format;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author findepi
+ */
+public class TrinoContainerTest {
+
+    @Test
+    public void testSimple() throws Exception {
+        try (TrinoContainer trino = new TrinoContainer(TrinoTestImages.TRINO_TEST_IMAGE)) {
+            trino.start();
+            try (Connection connection = trino.createConnection();
+                 Statement statement = connection.createStatement();
+                 ResultSet resultSet = statement.executeQuery("SELECT DISTINCT node_version FROM system.runtime.nodes")) {
+                assertTrue("No result", resultSet.next());
+                assertEquals("Trino version", TrinoContainer.DEFAULT_TAG, resultSet.getString("node_version"));
+            }
+        }
+    }
+
+    @Test
+    public void testSpecificVersion() throws Exception {
+        try (TrinoContainer trino = new TrinoContainer(TrinoTestImages.TRINO_PREVIOUS_VERSION_TEST_IMAGE)) {
+            trino.start();
+            try (Connection connection = trino.createConnection();
+                 Statement statement = connection.createStatement();
+                 ResultSet resultSet = statement.executeQuery("SELECT DISTINCT node_version FROM system.runtime.nodes")) {
+                assertTrue("No result", resultSet.next());
+                assertEquals("Trino version", TrinoTestImages.TRINO_PREVIOUS_VERSION_TEST_IMAGE.getVersionPart(), resultSet.getString("node_version"));
+            }
+        }
+    }
+
+    @Test
+    public void testQueryMemoryAndTpch() throws SQLException {
```

Comment:
this test doesn't seem to be testing anything Testcontainers related, I suggest we remove it to reduce the load on our CI

Reply:
it tests that all thing work as expected and that memory and tpch connectors are loaded & operational, this is important from consumer perspective, as these catalogs are most useful for tests

Reply:
but that's a property of the image, not our "wrapper" for it.

We take our CI very sensitively, as the execution time grows very fast (because we need to test real things, not just a couple of unit tests), and are trying to limit the scope of the tests to the things affected by our code, while not testing container's behaviour. I hope you will understand.